game.js

```diff
@@ -9,46 +9,29 @@ Game.prototype.play = function(){
 	debugger;
 	currentPlayer=this.currentPlayer;
 	if(currentPlayer===0){
-		randomOrder[0].placeMarker();
+		randomOrder[0].placeMarker(this);
 		currentPlayer=currentPlayer+1;
 	}
 	else{
-		randomOrder[1].placeMarker;
+		randomOrder[1].placeMarker(this);
 		currentPlayer=currentPlayer-1;
 	}
 }
 //Human Object
-function Player1(){
-	function placeMarker(){
-		var select = prompt("Select your position (1-9)");
-	//Validate number is between 1 and 9
-	if((select>=1)&&(select<=9)){
-		debugger;
-		console.log('valid input');
-		//Set the marker here by filling array
-	}
-	else
-	{
-		console.log("invalid input");
-		selectPosition();
-	}
-}
+function HumanPlayer(){
+	// Javascript equivalent to `def initialize`
```

Comment:
Instead of creating a Player1 `class`, think of these as being "types", so you need a "HumanPlayer" type/class of player, and a "ComputerPlayer" type/class of player.